examples/13-streamlines/00_plot_3d_streamlines.py

```diff
@@ -0,0 +1,207 @@
+# Meshed region is used as geometric base to compute the streamlines.
+# Velocity data is used to compute the streamlines. The velocity data must be nodal.
+
+###############################################################################
+# Get the meshed region:
+meshed_region = m_fluent.metadata.meshed_region
+
+###############################################################################
+# Get the velocity result at nodes:
+velocity_op = m_fluent.results.velocity()
+fc = velocity_op.outputs.fields_container()
+field = dpf.operators.averaging.to_nodal_fc(fields_container=fc).outputs.fields_container()[0]
+
+###############################################################################
+# Compute and plot the streamlines adjusting the request
+# ------------------------------------------------------
+# The following steps show you how to create streamlines using DpfPlotter, with several sets
+# of parameters. It demonstrates the issues that can happen and the adjustments that you can make.
+
+###############################################################################
+# First, a DpfPlotter is created and the streamline is created with default values.
+pl0 = DpfPlotter()
+try:
+    streamline_obj = compute_streamlines(meshed_region=meshed_region, field=field)
+    pl0.add_mesh(meshed_region=meshed_region, opacity=0.3)
+    pl0.add_streamlines(streamlines=streamline_obj)
+    pl0.show_figure(show_axes=True)
+except:
+    # It throws an error and ends here, because source points are not set correctly.
+    # To fix this issue, the source_center parameter is used to move
+    # the streamlines source center.
+    pass
+
+###############################################################################
+# Then, you can correctly set the source coordinates using the
+# "source_center" argument that moves the source center,
+# the "return_source" arguments that displays the source, and
+# "permissive" option that allows you to display the source even, if the computed
+# streamline size is zero.
+streamline_obj, source_obj = compute_streamlines(
+    meshed_region=meshed_region,
+    field=field,
+    return_source=True,
+    source_center=(0.1, 0.1, 0.2),
+)
+pl1 = DpfPlotter()
+pl1.add_mesh(meshed_region=meshed_region, opacity=0.3)
+pl1.add_streamlines(
+    streamlines=streamline_obj,
+    source=source_obj,
+    permissive=True,
+)
+pl1.show_figure(show_axes=True)
+
+###############################################################################
+# After the adjustment, the correct values for the "source_center" argument are set.
+# You can remove the "permissive" option.
+# You can display velocity data with a small opacity value to avoid hiding the streamlines.
+# More settings are added to adapt the streamlines creation to the geometry and
+# the data of the model:
+# - radius: streamlines radius
+# - n_points: source number of points
+# - source_radius
+# - max_time: allows the streamline to be computed along a certain length
```

Comment:
@anslpa how is this "time" linked to a length value? How do I know what max_time to put?

Reply:
@rafacanton, could you please help me to make that clearer ?

Reply:
@PProfizi Because streamlines are the first integrals of velocity. What we are doing is solving for this system of 3 ODEs:

```
dx/dt = v_x(x,y,z)
dy/dt = v_y(x,y,z)
dz/dt = v_z(x,y,z)
```

for an initial condition (x0,y0,z0). This integration will finish if:

- v->0 (the default value for this according to pyVista doc is 1e-12.
- the streamline leaves the domain.
- You integrate for a certain amount of time. The max_time controls this. If it is too short, the streamñines will be very short

Reply:
So I would maybe rephrase as:

- max_time: maximum integration time of the streamline. It controls the streamline length.

Reply:
Done, thank you!

Reply:
@rafacanton ah ok, my mistake was thinking that a streamline is computed on a snapshot velocity field at a specific instant, hence no time. However moving along this snapshot flow is still equivalent to moving in "time" like a particle would.